modules/20-arithmetics/80-linting/description.en.yml

```diff
@@ -1,7 +1,49 @@
 ---
 
 name: Linter
-theory: 
-instructions: 
-tips:
-definitions:
+theory: |
+  Since we’ve learned how to write simple programs, let's talk about the process of writing them.
+
+  Program code should be organized in a certain way so that it is clear and easy to be supported. Special sets of rules - standards - describe different aspects of code writing. The most common standard in JavaScript is [AirBnb](https://github.com/airbnb/javascript).
+
+  In any programming language there are utilities, the so-called **linters**. They check the code up to the standards. For example, [ESLint](https://eslint.org/) analyzes JavaScript code to flag errors.
+
+  Take a look at the example from the previous lesson:
+
+  ```javascript
+  console.log(8/2+5 - -3 / 2); // => 10.5
+  ```
+
+  Linter won’t be happy about it, because of the violation of several rules:
+
+    * [space-infix-ops](https://eslint.org/docs/rules/space-infix-ops) – No space between operator and operands.
+    * [no-mixed-operators](https://eslint.org/docs/rules/no-mixed-operators) – You cannot write code with different operations used in the same expression without putting parentheses.
+
+  In the last lesson we recognized that such an abundance of numbers and symbols may be confusing, and decided to add parentheses just for the sake of readability:
+
+  ```javascript
+  console.log(((8 / 2) + 5) - (-3 / 2)); // => 10.5
+  ```
+
+  This code does not violate the rules, and the linter will «keep silent».
+
+  In the instructions of the previous lesson, you probably did this:
+
+  ```javascript
+  console.log(70 * (3 + 4) / (8 + 2));
+  ```
+
+  Is there any violation of standard here?
+
+  Unfortunately, yes. This time, operations `*` and `/` are in the same expression and parentheses are missing . You can solve this problem by adding additional parentheses. But at some point, too many parentheses can make the code incomprehensible again. At this point you can divide the expression into separate parts. We will learn how to do this in the following lessons.
+
+  [no-mixed-operators](https://eslint.org/docs/rules/no-mixed-operators) — is an example of many rules. Other ones describe indentations, entity names, parentheses, mathematical operations, line length, and many other aspects. Each individual rule seems rather small, not very important. But together they form the basis of a good code.
```

Comment:
Each individual rule seems rather small, not very important. But together => Each rule seems small and insignificant, but together they